CitationStyleLanguageSettingsForm.inc.php

```diff
@@ -29,6 +32,13 @@ public function __construct($plugin) {
 		$this->plugin = $plugin;
 		$this->addCheck(new FormValidatorPost($this));
 		$this->addCheck(new FormValidatorCSRF($this));
+		if ($this->plugin->isApplicationOmp()) {
+			$request = Application::get()->getRequest();
+			$context = $request->getContext();
+			$contextId = $context ? $context->getId() : 0;
+			$chapterPlugin = PluginRegistry::getPlugin('generic', 'chapterfrontendpageplugin');
+			$this->isChapterFrontendPagePluginEnabled = NULL !== $chapterPlugin && $chapterPlugin->getEnabled($contextId);
```

Comment:
I'm trying to remember, but I think you might run into timing issues here, because I think the order that plugins are registered can change. So you may run into cases where the CSL plugin is running its constructor before the chapter frontend plugin has been registered.